### ISSUE TITLE
Standard install directory structure

### ISSUE THREAD

**Issue by matteodelabre** [matteodelabre/toltec#254]
There is currently no guidance as to what files must be installed in which directories on the device. For example, we have packages that install binaries to `/opt/lib`, there are files both in `/opt/usr/share` and `/opt/share`, some packages install files to the root that should go to `/opt` instead.

We should agree on a structure and enforce it through lint checks.

**Comment by Eeems** [comment]
To simplify things we should probably start with a standard structure already defined by arch or debian and just transpose it inside /opt

**Comment by matteodelabre** [comment]
Agreed. I think we should also seek compatibility with the structure of Entware packages (if they do have a consistent structure).

**Comment by LinusCDE** [comment]
Also agreed. We had this problem earlier though I guess. One example are the document readers that would normally go into `/opt`. I install plato to `/opt/lib/plato` (was the best way of an opt-like directory imo) while koreader goes into `/opt/koreader` if I'm not mistaken.

Problem was that `/opt/opt` was less ideal because of chroot possibilities if not mistaken.
